src/frontends/onnx/frontend/src/utils/tensor_external_data.cpp

```diff
@@ -36,6 +36,17 @@ TensorExternalData::TensorExternalData(const TensorProto& tensor) {
 
 Buffer<ov::MappedMemory> TensorExternalData::load_external_mmap_data(const std::string& model_dir,
                                                                      MappedMemoryHandles cache) const {
+    if (m_data_location == "*/_ORT_MEM_ADDR_/*") {
```

Comment:
I don't like this "magic number", is it documented somewhere?

Reply:
Hmmm... I see https://github.com/microsoft/onnxruntime/blob/4f6ae14e09729b3e3aba921de2e5bcc26d3e7768/onnxruntime/core/framework/tensorprotoutils.h#L206
Could you please create a constant with comment and place a link on original definition?